### PR TITLE
New water meter types

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

**Comment by evgeniya-engel** [comment]
@shambergoldstein hey Shamber, it's ready for your review!

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 1]
```diff
@@ -256,6 +257,7 @@ descriptor:
   user: "A person who uses or operates something."
   vapor: "A substance in the gas phase at a temperature lower than its critical temperature."
   ventilation: "Process used to provide fresh (typically atmospheric) air into a system or zone."
+  waste: "Substance generated after the use in a variety of deliberate applications or processes."
```
Comment (shambergoldstein) [inline ontology/yaml/resources/subfields/subfields.yaml:260]: reword to "An undesirable substance gathered in a fluid as it goes through deliberate applications or processes"

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 2]
```diff
@@ -209,6 +209,7 @@ descriptor:
   protection: "Act of preventing damage to object."
   purge: "The act of removing impurities or contaminants from a space or system during particular phases of operation. For example, when purging a zone during morning warm up using additional ventilation air."
   rain: "Liquid water in the form of droplets that have condensed from atmospheric water vapor."
+  recirculation: "A process of moving substance through a system or loop more than once to achieve a desired condition."
```
Comment (shambergoldstein) [inline ontology/yaml/resources/subfields/subfields.yaml:212]: nit: change to "moving a substance"

**Review by shambergoldstein — CHANGES_REQUESTED** [review]

**Comment by evgeniya-engel** [comment]
@shambergoldstein addressed your comments, thank you!

**Review by shambergoldstein — APPROVED** [review]

**Pull request merged**